HelgaShiryaeva/assignment3/DictionaryTest.java

```diff
@@ -0,0 +1,61 @@
+package assignment3;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.HashSet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DictionaryTest {
+    private Dictionary dictionary;
+    private WordSearch wordSearch;

```

Comment:
`WordSearch` should have its own test class. Idea behind unit tests is that they test a single unit of code, which in case of Java is typically a single class.